server/server.go

```diff
@@ -65,6 +71,17 @@ func New(cfg Config) (*Server, error) {
 	return ret, nil
 }
 
+// Metrics returns a collection of metrics for s. THe caller is responsible for
+// publishing the result to the metrics exporter.
+func (s *Server) Metrics() expvar.Var {
+	m := new(expvar.Map)
```

Comment:
this one can remain a regular map, its a container of unrelated metrics.

Reply:
I turned it into a metrics.Set